app/assets/javascripts/admin/components/report-filters/include-subcategories.js

```diff
@@ -0,0 +1,23 @@
+import FilterComponent from "admin/components/report-filters/filter";
+import discourseComputed from "discourse-common/utils/decorators";
+
+export default FilterComponent.extend({
+  checked: false,
+
+  didReceiveAttrs() {
+    this.set("checked", this.filter.default === "true");
+  },
+
+  @discourseComputed()
```

Comment:
would prefer using direct ember computed here, you don't use anything discourseComputed macro brings